sample/kotlin/src/main/kotlin/com/datadog/android/sample/server/LocalServer.kt

```diff
@@ -38,7 +38,7 @@ class LocalServer {
 
     fun stop() {
         Thread {
-            engine?.stop(SHUTDOWN_MS, STOP_TIMEOUT_MS, TimeUnit.MILLISECONDS)
```

Comment:
this argument is gone with the referenced version of `ktor`, now this method can take only 2 arguments.